readme.md

```diff
@@ -1,19 +1,9 @@
 # Hello world
```

Comment:
The problem is that the student is trying to merge on Mate's develop branch, when she should have opened the PR from his develop into mate`s main

the branch in the pull request needs to be changed
https://mate-academy.github.io/layout_task-guideline/#create-a-pull-request-pr